ospfd/ospf_sr.c

```diff
@@ -1058,26 +1117,10 @@ void ospf_sr_ri_lsa_update(struct ospf_lsa *lsa)
 		return;
 	}
 
-	/* Get SR Node in hash table from Router ID */
-	srn = hash_get(OspfSR.neighbors, (void *)&(lsah->adv_router),
-		       (void *)sr_node_new);
-
-	/* Sanity check */
-	if (srn == NULL) {
-		flog_err(EC_OSPF_SR_NODE_CREATE,
-			 "SR (%s): Abort! can't create SR node in hash table",
-			 __func__);
-		return;
-	}
+	/* Search SR Node in hash table from Router ID */
+	srn = (struct sr_node *)hash_lookup(OspfSR.neighbors,
+					    (void *)&(lsah->adv_router));
```

Comment:
nit: unnecessary explicit cast to `void *`

Reply:
done